basin3d/core/schema/query.py

```diff
@@ -61,6 +63,14 @@ class Config:
         # Validate all fields when initialized
         validate_all = True
 
+    def get_mapped_fields(self) -> list:
+        """Get the query fields that have mappings. Subclasses may overwrite this base function"""
+        return []
+
+    def get_prefixed_fields(self) -> list:
+        """Get the query fields that have prefixes. Subclasses may overwrite ths base function"""
+        return []
+
```

Comment:
This should be a field that is a pydantic ClassVar instead

```suggestion
   prefixed_fields: ClassVar[List[str]] = []

```

Reply:
Done. Changed get_mapped_fields similarly.